lib/Conch/Controller/Schema.pm

```diff
@@ -46,6 +46,22 @@ sub get ($c) {
 	}
 
 	my $schema = $validator->get("/definitions/$name");
+
+    my sub inline_ref ( $ref, $schema ) {
```

Comment:
This section (filling in the $refs) doesn't really go with the commit as it is currently labelled, but ok...

Reply:
Split this out into it's own commit.